conf/deployuidb.sh

```
@@ -7,24 +7,59 @@ host="$3"
 EOF
+fi
 
-mysql --user=$user --password=$password --host=$host --port=$port << EOF
-grant usage on *.* to 'root'@'localhost';
-grant usage on *.* to 'root'@'%';
-DROP DATABASE IF EXISTS zstack_ui;
-CREATE DATABASE zstack_ui;
-grant all privileges on zstack_ui.* to root@'%' identified by "$password";
-grant all privileges on zstack_ui.* to root@'localhost' identified by "$password";
-flush privileges;
+if command -v greatdb &> /dev/null; then
```

Shellcheck (0.10.0)

[warning] In POSIX sh, &> is undefined.

(SC3020)